Assets/Scripts/Audio/AudioController.cs

```diff
@@ -0,0 +1,106 @@
+﻿using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+
+
+public class AudioController : MonoBehaviour {
```

Comment:
yea wasn't able to come up with a better name, sure I can rename it on my next patch